src/app/main.cpp

```diff
@@ -405,7 +405,9 @@ void myMessageOutput( QtMsgType type, const QMessageLogContext &, const QString
            msg.contains( QLatin1String( "QGestureManager::deliverEvent" ), Qt::CaseInsensitive ) ||
            msg.startsWith( QLatin1String( "libpng warning: iCCP: known incorrect sRGB profile" ), Qt::CaseInsensitive ) ||
            msg.contains( QLatin1String( "Could not add child element to parent element because the types are incorrect" ), Qt::CaseInsensitive ) ||
-           msg.contains( QLatin1String( "OpenType support missing for" ), Qt::CaseInsensitive ) )
+           msg.contains( QLatin1String( "OpenType support missing for" ), Qt::CaseInsensitive ) ||
+           msg.contains( QLatin1String( "Wayland does not support" ), Qt::CaseInsensitive ) ||
```

Comment:
Qt5 only?

Reply:
Not sure, but there's a range of messages that are shown like "Wayland does not support QWidget::raise", etc. basically just noise, we can't fix those. 

I'd run x11 if I could, but fedora 40 KDE removed that . There's no winners here 🤷‍♂️